test/values.ts

```diff
@@ -458,7 +460,9 @@ export const makeExchangeRates = (
   },
 })
 
-export const mockTokenBalances = {
+export const mockTokenBalances: Record<string, StoredTokenBalance> = {
+  // NOTE: important to keep 'symbol' fields in this object matching their counterparts from here: https://github.com/valora-inc/address-metadata/blob/main/src/data/mainnet/tokens-info.json ,
+  //  particularly for CICO currencies
   '0x00400FcbF0816bebB94654259de7273f4A05c762': {
```

Comment:
Should all addresses in this list be lowercase?

Reply:
fixed